fe/fe-core/src/main/java/com/starrocks/common/profile/TracerImpl.java

```diff
@@ -209,9 +219,18 @@ public void buildVars(RuntimeProfile parent) {
         }
     }
 
+    private void buildReasons(RuntimeProfile profile) {
```

Comment:
need print reason in `printLogs`? otherwise can't get `reason` though the command